src/components/10-atoms/text/index.js

```diff
@@ -28,59 +28,16 @@ class AXAText extends NoShadowDOM {
     applyDefaults(this);
   }
 
-  watch(mode) {
-    switch (mode) {
-      case 'stop':
-        if (this._observer) {
-          this._observer.disconnect();
-        }
-        break;
-      case 'start':
-        if (!this._observer) {
-          this._observer = new MutationObserver(() => this.customRender());
-        }
-        this._observer.observe(this, {
-          childList: true,
-        });
-        break;
-      default:
-        break;
-    }
-  }
-
-  // we can't use 'render' since this has special semantics under lit-element and its
-  // documentation mandates it should return a lit-element TemplateResult,
-  // while we want to manipulate DOM ourselves here for good reasons.
-  customRender() {
-    // do we have 'pure', non-empty text, e.g. <axa-text>Hello, World</axa-text>?
-    const nonWhitespaceTextNodes = [...this.childNodes].filter(
-      node => node.nodeType === TEXT_NODE_TYPE && node.textContent.trim()
-    );
-    if (nonWhitespaceTextNodes.length) {
-      // yes, wrap it in 1 <p> node to keep screenreaders happy (they offer
-      // paragraph-to-paragraph-jumping keyboard shortcuts that work by detecting <p>s):
-      const paragraph = document.createElement('p');
-      paragraph.textContent = this.textContent;
-      // pause child-update watcher
-      this.watch('stop');
-      // delete the children, now that their content has been harvested
-      this.innerHTML = '';
-      // add the <p> node
-      this.appendChild(paragraph);
-      // restart child-update watcher
-      this.watch('start');
-    }
-  }
-
   connectedCallback() {
     super.connectedCallback();
-    this.customRender();
+
+    if (!this.tagless) {
+      this.setAttribute('aria-role', 'paragraph');
+    }
   }
 
-  disconnectedCallback() {
-    super.disconnectedCallback();
-    // remove installed observer
-    this.watch('stop');
+  render() {
+    return this.tagless ? html`<slot></slot>` : html`<p><slot></slot></p>`;
```

Comment:
DRYify more? Avoid double slot mention.

Reply:
I had tried, but it was not legible. Therefore, I accept the small redundant one.